### PR TITLE
Fixed transparency for LaserZap

### PR DESCRIPTION
**Pull request by Mailaender**
This now allows to set initial transparency values, but still does the fade-out effect. The documentation tag was out-dated. We may be better off explaining value types at a separate place instead of individually.

### CODE REVIEW

File: OpenRA.Mods.Common/Projectiles/LaserZap.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ public class LaserZapInfo : IProjectileInfo
 
 		public readonly bool UsePlayerColor = false;
 
-		[Desc("Laser color in (A,)R,G,B.")]
+		[Desc("Laser color.")]
```
Comment (pchote) [inline OpenRA.Mods.Common/Projectiles/LaserZap.cs]: Having the description for "Laser: Color" be "Laser color." doesn't tell us anything useful.  Just remove it.

Comment (Mailaender) [inline OpenRA.Mods.Common/Projectiles/LaserZap.cs]: @Phrohdoh will probably disagree.

**Comment by GraionDilach** [comment]
:+1:

**Comment by reaperrr** [comment]
:+1:

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/c2e27ccbc68193f12d4a8186484585f4325345b8...1115b7424b3d838b49d0e360406f76778f16b995)

**Comment by Mailaender** [comment]
> We may be better off explaining value types at a separate place instead of individually.

https://github.com/OpenRA/OpenRA/pull/11913 adds explanations for the color notation.

**Pull request merged**